dev-java/openjfx/openjfx-17.0.0.1_p1.ebuild

```diff
@@ -0,0 +1,234 @@
+# Copyright 2019-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+MY_PV="${PV/_p/+}"
+MY_PV2="${MY_PV%+*}"
+SLOT="${MY_PV%%[.+]*}"
+
+# https://docs.gradle.org/current/userguide/compatibility.html
+EGRADLE_VER="7.3"
+
+inherit flag-o-matic java-pkg-2 multiprocessing toolchain-funcs
+
+DESCRIPTION="Java OpenJFX client application platform"
+HOMEPAGE="https://openjfx.io"
+
+SRC_URI="
+	https://github.com/openjdk/jfx/archive/refs/tags/${MY_PV}.tar.gz -> ${P}.tar.gz
+	https://downloads.gradle.org/distributions/gradle-${EGRADLE_VER}-bin.zip
+	https://repo.maven.apache.org/maven2/org/apache/lucene/lucene-sandbox/7.7.3/lucene-sandbox-7.7.3.jar
+	https://repo.maven.apache.org/maven2/org/apache/lucene/lucene-grouping/7.7.3/lucene-grouping-7.7.3.jar
+	https://repo.maven.apache.org/maven2/org/apache/lucene/lucene-queryparser/7.7.3/lucene-queryparser-7.7.3.jar
+	https://repo.maven.apache.org/maven2/org/apache/lucene/lucene-queries/7.7.3/lucene-queries-7.7.3.jar
+	https://repo.maven.apache.org/maven2/org/apache/lucene/lucene-core/7.7.3/lucene-core-7.7.3.jar
```

Comment:
`lucene-*-7.7.3.jar` should be packaged from source and added to `dev-java/lucene` as a version bump:
```
SRC_URI="https://archive.apache.org/dist/${PN}/java/${PV}/${P}-src.tgz
    verify-sig? ( https://archive.apache.org/dist/${PN}/java/${PV}/${P}-src.tgz.asc )"
```

Reply:
ideally yes, but I have used the same strategy as current openjfx packages